packages/corvid-cli/src/apps/clone.js

```diff
@@ -75,10 +76,17 @@ async function extractMetasiteIdAndName(url, cookie) {
 async function clone(spinner, args, cookie) {
   spinner.start(chalk.grey(getMessage("Clone_Getting_Site_Data")));
   try {
-    const { metasiteId } = await extractMetasiteIdAndName(args.url, cookie);
+    const { metasiteId, siteName } = await extractMetasiteIdAndName(
+      args.url,
+      cookie
+    );
+
+    if (siteName == null) {
```

Comment:
i don't understand the distinction between a case where siteName is null and when metasiteId is null.
is it possible to get a site with no name but with an id ?

Reply:
if it's a public url, metasiteId will be taken from it directly, without checking ownership

Reply:
if there is no siteName, user is definitely not own this site

Reply:
i still don't understand.
if the given url matches a site in the list returned from the server, that matched site will have both a name and a metasiteId.
let's talk f2f